transport/tchannel/channel_outbound.go

```diff
@@ -74,6 +74,12 @@ type ChannelOutbound struct {
 	once *lifecycle.Once
 }
 
+// TransportName is the transport name that will be set on `transport.Request`
+// struct.
+func (o *ChannelOutbound) TransportName() string {
```

Comment:
Could you please add a unit-test for this function? I understand that it's pretty straightforward, but still.

(Optional, you may skip it): probably it makes sense to add a compile-check assert, like
```
var _ transport.Namer = (*ChannelOutbound)(nil)
```

Reply:
Added interface compliance check and a unit test.